src/java/org/apache/cassandra/net/InboundConnectionSettings.java

```diff
@@ -83,8 +83,8 @@ public boolean authenticate(InetAddress address, int port)
 
     public String toString()
     {
-        return format("address: (%s), nic: %s, encryption: %s",
-                      bindAddress, FBUtilities.getNetworkInterface(bindAddress.address), SocketFactory.encryptionLogStatement(null, encryption));
+        return format("address: (%s), nic: %s, encrypted: %s",
```

Comment:
wasn't encryption correct before?  you are doing a summary and not a boolean; the following feels weird

```
encrypted: unencrypted
```

Reply:
bah, fail putting it back the way it was before a comment from Dinesh. Will fix.